<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1110</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1110</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>151</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_XRotation">
     <item>
      <widget class="QLabel" name="label_XRotation">
       <property name="text">
        <string>X rotation degree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_XRotation">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>151</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_YRotation">
     <item>
      <widget class="QLabel" name="label_YRotation">
       <property name="text">
        <string>Y rotation degree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_YRotation">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDial" name="dial_XRotation">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>45.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial_YRotation">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>290</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>45.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_restart">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>690</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>49</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>151</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_ZRotation">
     <item>
      <widget class="QLabel" name="label_ZRotation">
       <property name="text">
        <string>Z rotation degree</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_ZRotation">
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDial" name="dial_ZRotation">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>510</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>45.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_options">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>120</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QSlider" name="verticalSlider_zoom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>16</width>
       <height>221</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>107</y>
       <width>51</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>Zoom</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>39</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_options">
     <item>
      <widget class="QCheckBox" name="checkBox_XRay">
       <property name="text">
        <string>X-Ray</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_colors">
       <property name="text">
        <string>Face colors</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
